quesma/schema/registry.go

```diff
@@ -201,7 +202,15 @@ func (s *schemaRegistry) populateSchemaFromTableDefinition(definitions map[strin
 				// if field encodings are not coming from ingest e.g. encodings map
 				// is empty, read them from persistent storage, e.g. column comment
 			} else if len(column.Comment) > 0 {
-				propertyName = FieldName(column.Comment)
+				propertyName = FieldName(column.Name)
+				metadata, err := comment_metadata.UnmarshallCommentMetadata(column.Comment)
+				if err != nil {
+					logger.Warn().Msgf("error unmarshalling column '%s' (table: %s)  comment metadata: %s %v", indexName, column.Name, column.Comment, err)
+				} else {
+					if fieldName, ok := metadata.Values[comment_metadata.ElasticFieldName]; ok {
```

Comment:
I know this becomes a little if-ology but do if unmarshalling went fine and `metadata.Values` doesn't contain  `comment_metadata.ElasticFieldName` should we log this as something **really** strange ?

to reduce if's maybe you could use
```go
if metadata, err := comment_metadata.UnmarshallCommentMetadata(column.Comment); err == nil {
```

Reply:
Now that I think about it - perhaps we should provide a little facade method and let this logic live in
`metadata.getElasticsearchFieldName` ? 🤔 
Just a suggestion, not really a request.

Reply:
We don't keep in a schema any issues occurred while reading the table definition. In this case, if we cannot parse the metadata, we will expose that column with the default name. This can be confusing for the end user. 

We should log a warning if the format is invalid only. I'll change it.